src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -376,26 +375,6 @@ def load_command_table(self, _):
         g.command('enable', 'enable_recommendation')
         g.command('disable', 'disable_recommendation')
 
-    database_threat_detection_policies_operations = CliCommandType(
-        operations_tmpl='azure.mgmt.sql.operations#DatabaseSecurityAlertPoliciesOperations.{}',
-        client_factory=get_sql_database_threat_detection_policies_operations)
-
-    database_threat_detection_policy_update_sdk = CliCommandType(
-        operations_tmpl='azure.cli.command_modules.sql.custom#{}')
-
-    with self.command_group('sql db threat-policy',
-                            database_threat_detection_policies_operations,
-                            client_factory=get_sql_database_threat_detection_policies_operations,
-                            deprecate_info=self.deprecate(redirect='sql db advanced-threat-protection-setting', hide=True, expiration='2.45.0')) as g:
```

Comment:
Can you please create another PR to modify the expiration version to `2.49.0`? Keeping `2.45.0` in our code will cause build failure of next CLI release...